### PR TITLE
Release 2019.8.20

### PR DESCRIPTION
**Pull request by theacodes**

### CODE REVIEW

**Comment by dhermes** [comment]
👏

File: setup.py [hunk 1]
```diff
@@ -22,15 +22,15 @@
...
+    version="2019.8.20",
     description="Flexible test automation.",
     long_description=long_description,
     url="https://nox.thea.codes",
     author="Alethea Katherine Flowers",
     author_email="me@thea.codes",
     license="Apache Software License",
     classifiers=[
-        "Development Status :: 4 - Beta",
+        "Development Status :: 5 - Production/Stable",
```
Comment (dhermes) [inline setup.py]: 😮

**Comment by theacodes** [comment]
The classifier change was *long* overdue. We were stable when I switched to
calver.

**Comment by dhermes** [comment]
Totes. Congrats all!

**Pull request merged**